xai_compare/datasets.py

```diff
@@ -0,0 +1,166 @@
+# ----------------------------------------------------------------------------------------------------
+# Module datasets
```

Comment:
what about the FICO dataset that we've uploaded to the data dir?

Reply:
We added the ability to import a FICO dataset, as discussed.